fastestimator/trace/metric/recall.py

```diff
@@ -72,7 +79,23 @@ def on_batch_end(self, data: Data) -> None:
 
     def on_epoch_end(self, data: Data) -> None:
         if self.binary_classification:
-            score = recall_score(self.y_true, self.y_pred, average='binary')
+            score = recall_score(self.y_true, self.y_pred, average='binary', **self.kwargs)
         else:
-            score = recall_score(self.y_true, self.y_pred, average=None)
+            score = recall_score(self.y_true, self.y_pred, average=None, **self.kwargs)
         data.write_with_log(self.outputs[0], score)
+
+    @staticmethod
+    def check_kwarg(kwargs: Dict[str, Any]) -> None:
```

Comment:
should these all be named 'check_kwargs' instead of kwarg?

Reply:
fixed